google/codelab/provision_spinnaker_codelab_image.sh

```diff
@@ -58,6 +58,7 @@ sudo -u jenkins -H sh -c '/home/jenkins/aptly publish repo -architectures="amd64
 
 # as jenkins, configure jenkins config directory
 # this storage bucket is public so we can pull the jenkins config from anywhere
+cd /var/lib/jenkins
```

Comment:
Yes, this fixes a bug I introduced with the previous PR. The Jenkins config and `hello-karyon-rxnetty` git repo go in /var/lib/jenkins and the aptly executable and repo goes in /home/jenkins. This change has been tested from start to finish automating the codelab image.